### ISSUE TITLE
RSS readers comparison 

### ISSUE THREAD

**Issue by shuvashish76** [MuntashirAkon/android-debloat-list#48]
|  | Features | [Feeder](https://github.com/spacecowboy/Feeder) | [Read You](https://github.com/Ashinch/ReadYou) |
|--------|--------|--------|--------|
| 1 | F-Droid (last update) status | 2024-01-30 | 2024-01-22 |
| 2 | Size (on F-Droid) | 13MB | 4.9MB |
| 3 | App Store (F&G) | ✅ | F |
| 4 | OPML | v1.1 | v2.0 |
| 5 | Export/Import | ✅ | ✅ |
| 6 | Accounts | Only local | Local+Services+Self-hosted |
| 7 | Export/Import | ✅ | ✅ |
| 8 | Saved/Starred articles | ✅ | ✅ |
| 9 | Block list | ✅ | ❌ |
| 10 | Export/Import | ✅ | ✅ |
| 11 | Parse full content| ☑️ (only for feeds)| ✅ (for groups & feed) |
| 12 | Extra Unique IDs | ✅ | ❌ |
| 13 | Tags | ❌ | ❌ |
| 14 | Swipe action | ✅ | ☑️ |
| 15 | Switch articles | ❌ | ☑️ |
| 16 | View & save Images | ❌ | ❌ |
| 17 | Tags | ❌ | ❌ |
| 18 | Notification actions | ✅ | ❌ |
| 19 | Notification actions | ✅ | ❌ |
| 20 | Archive | 50-1000 items per feed | 1month-always |
| 21 | Clear articles | ❌ | ✅ |
| 22 | Notification | ☑️ (for specific feeds only) | ✅ (for feeds & as group) |
| 23 | Muted feeds | ❌ | ❌ |
| 24 | Read aloud | ✅ | ❌ |
| 25 | Notification actions | ✅ | ❌ |
| 26 | Estimated reading time | ✅ | ❌ |
| 27 | Manual fetch articles | ✅ (for feeds & groups) | ❌  (only All)|
| 28 | Tags | ❌ | ❌ |
| 29 | Widget | ❌ | ❌ |
| 30 | Sorting | ❌ | ❌ |
| 31 | Search | ❌ | ❌ |
| 32 | Thumbnail preview | ✅ | ✅ |
| 33 | Sub-groups | ❌ | ❌ |

**Comment by MuntashirAkon** [comment]
At this point, they should get some ideas from [NetNewsWire](https://netnewswire.com/).

**Comment by shuvashish76** [comment]
The devs taking ideas mostly from https://reederapp.com/ since they're using this application on their personal device.

**Comment by MuntashirAkon** [comment]
> The devs taking ideas mostly from https://reederapp.com/ since they're using this application on their personal device.

Yes, I know about this $10 app. It's not much of a different than the free NetNewsWire (which is open source) except for the cloud sync support (NNW only supports iCloud sync) and bionic reading (which IMO quite useless).
